src/main/webapp/app/app-routing.module.ts

```diff
@@ -116,6 +117,11 @@ const LAYOUT_ROUTES: Routes = [navbarRoute, ...errorRoute];
                     path: 'lti',
                     loadChildren: () => import('./lti/lti.module').then((m) => m.ArtemisLtiModule),
                 },
+                {
+                    path: 'chat',
```

Comment:
Three things:
1. This may be a bit too general of a name. Using "iris-chat" might be better
2. How do you pass the information about which iris session to use into here? What happens if the user has two sessions open at one time?
3. What happens if a user opens this URL manually?

Reply:
1. Name will be changed
2. The iris session will be added to the url and then will be controlled in component's ts file. However the group was not sure about handling the behaviour of the new window, that's why we left it.
3. They will not be able to open it in the future if we add session Id too, I assume?

Reply:
Update: We will remove this part completely as Patrick did not want a new window but only a full size widget instead.